packages/ui-extensions/src/surfaces/checkout/api/address-autocomplete/shared.ts

```diff
@@ -29,19 +29,20 @@ export interface AddressAutocompleteSuggestion {
   id?: string;
 
   /**
-   * If provided, this will be used to highlight the substrings
+   * A list of substrings that matched the original search query.
+   *
+   * If `matchedSubstrings` are provided, then they will be used to highlight the substrings
    * of the suggestions that matched the original search query.
    *
    * @example [{offset: 0, length: 4}]
    */
   matchedSubstrings?: MatchedSubstring[];
 
   /**
-   * If provided, the autocomplete field will use the formatted address to
-   * auto-fill the remaining address fields.
+   * The formatted address to that will auto-fill the remaining address fields.
```

Comment:
This sentence doesn't look right 🤔 
```suggestion
   * The formatted address that will auto-fill the remaining address fields.
```

Reply:
Hmm yeah, maybe I won't include that change but we should change that in our private packages later.